MSAL/src/MSALAccount.m

```diff
@@ -184,4 +203,25 @@ - (BOOL)isEqualToUser:(MSALAccount *)user
     return result;
 }
 
+#pragma mark - Tenant profiles
+
+- (NSArray<MSALTenantProfile *> *)tenantProfiles
+{
+    return self.mTenantProfiles;
+}
+
+- (void)addTenantProfiles:(NSArray<MSALTenantProfile *> *)tenantProfiles
+{
+    if (tenantProfiles.count <= 0) return;
+    
+    if (self.tenantProfiles)
```

Comment:
nit: check for mTenantProfiles here as well? It's the same, but it's easier to read when it's checking and using the same property

Reply:
changed :)